repo/js/批量讨伐角色养成材料BOSS/BOSS配置编辑器.html

```
@@ -0,0 +1,1127 @@
+
+                <div class="form-group">
+                    <div class="checkbox-group">
+                        <input type="checkbox" id="returnToStatueAfterEachRound">
+                        <label for="returnToStatueAfterEachRound" style="display: inline; margin-bottom: 0;">每轮结束后回到七天神像回血</label>
+                    </div>
+                </div>
+
+                </div>
+            </div>
+        </div>
+    </div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)